src/prefect/tasks/monte_carlo/client.py

```diff
@@ -0,0 +1,256 @@
+                                warehouses {
+                                    name
+                                    uuid
+                                }
+                            }
+                        }
+                    }
+                    """
+        )
+        warehouses = response["data"]["getUser"]["account"]["warehouses"]
+        id_ = [w for w in warehouses if w["name"] == warehouse_name][0].get("uuid")
+        return id_
+
+    def get_resources(self) -> List[Dict[str, Any]]:
+        response = self.graphql(
+            query="""
+                    query {
+                        getResources {
+                            name
+                            type
+                            id
+                            uuid
+                            isDefault
+                            isUserProvided
+                        }
+                    }
+                    """
+        )
+        return response["data"]["getResources"]
+
+    def get_table_mcon(self, full_table_id: str, warehouse_name: str) -> str:
+        response = self.graphql(
+            query="""
+                    query($dwh_id: UUID!, $full_table_id: String!) {
+                        getTable(dwId: $dwh_id,
+                            fullTableId: $full_table_id) {
+                            mcon
+                        }
+                    }
+                    """,
+            variables=dict(
+                dwh_id=self.get_warehouse_id_by_name(warehouse_name),
+                full_table_id=full_table_id,
+            ),
+        )
+        return response["data"]["getTable"]["mcon"]
+
+    def get_node_mcon(self, object_id: str, object_type: str) -> str:
+        response = self.graphql(
+            query="""
+                    query($object_id: String!, $object_type: String!) {
+                        getLineageNode(objectId: $object_id,
+                            objectType: $object_type) {
+                            mcon
+                        }
+                    }
+                    """,
+            variables=dict(object_id=object_id, object_type=object_type),
+        )
+        mcon = response["data"]["getLineageNode"]["mcon"]
+        self.logger.info("Object_id %s has the following MCON: %s", object_id, mcon)
+        return mcon
+
+    def create_or_update_tags_for_mcon(
+        self, key: str, value: str, mcon: str
+    ) -> Dict[str, Any]:
+        response = self.graphql(
+            query="""
+                mutation($mcon_id: String!, $key: String!, $value: String!) {
+                    createOrUpdateObjectProperty(mconId: $mcon_id,
+                        propertyName: $key, propertyValue: $value) {
+                        objectProperty {
+                            id
+                        }
+                    }
+                }
+                """,
+            variables=dict(mcon_id=mcon, key=key, value=value),
+        )
+        return response
```

Comment:
In other mutations you return just the `mcon`. Would it make sense for consistency sake to return just the `objectProperty` from this function?